### PR TITLE
dev -> alpha

### PR DESCRIPTION
**Pull request by aermakov-zalando**
Highlights:
* k8s as flannel backend
* Disable image-policy simulation mode
* Fix source IP filtering in skipper
* HA kube-dns

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
👍

File: cluster/manifests/kube-dns/depl-kube-dns.yaml [hunk 1]
```diff
@@ -27,6 +27,18 @@ spec:
...
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: application
+                    operator: In
+                    values:
+                    - kube-dns-backup
```
Comment (linki) [inline cluster/manifests/kube-dns/depl-kube-dns.yaml]: This needs to be changed to `kube-dns-b` as well.

Comment (aermakov-zalando) [reply cluster/manifests/kube-dns/depl-kube-dns.yaml]: Fixed: https://github.com/zalando-incubator/kubernetes-on-aws/pull/812

**Comment by linki** [comment]
👍

**Comment by aermakov-zalando** [comment]
👍

**Comment by aermakov-zalando** [comment]
👎 
Let's merge #812 first.

**Comment by aermakov-zalando** [comment]
Let's merge #812 first.

**Comment by aermakov-zalando** [comment]
👍

**Comment by linki** [comment]
👍

**Pull request merged**